event_routing_backends/helpers.py

```diff
@@ -230,11 +229,9 @@ def make_course_url(course_id):
     Returns:
         str
     """
-    return '{root_url}{course_root_url}'.format(
+    return '{root_url}/course/{course_id}'.format(
```

Comment:
@rehanedly can we make a method in BaseTransformer to make IRIs of object
`get_object_iri(object_type, object_id)`

Reply:
@ziafazal done